### DISCUSSION TITLE
"Read more about" link sometimes not being generated, HTML broken

### DISCUSSION THREAD

**Discussion by FuadEfendi** [hbstack/.github#69] [Issues and bugs]
Hi Razon, could you please check this page, https://islamictermslexicon.com/categories/culture/

Some cards don't have "Read more ..." link and if you check HTML it is broken. But Hugo build doesn't report any issues. 

https://github.com/tokenizer-inc-canada/IslamicTermsLexicon.com/

Thanks,

**Comment by FuadEfendi** [comment]
I think some of my pages have `summary: ` in front matter that's why; when I remove "summary" issue disappear (and I have "Read more about ..." when I remove "summary" from front matter)
Still look suspicious...

**Comment by razonyang** [comment]
Should be fixed since [github.com/hbstack/blog@v0.37.1](https://github.com/hbstack/blog/releases/tag/v0.37.1), the read more button should be always shown if `description` is specified, otherwise determind by the page's `Truncated` read-only variable.

**Comment by FuadEfendi** [comment]
Thank you Razon, it works!

**Comment by razonyang** [comment]
Thanks for reporting.
